plugins/tours/tests/phpunit/integration/template/renderPostmetaBeforeContent.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ *  Tests for render_postmeta_before_content()
+ *
+ * @since      1.0.0
+ * @author     Robert A. Gadon
+ * @package    spiralWebDb\CornerstoneTours\Template\Tests\Integration
+ * @link       http://spiralwebdb.com
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Template;
+
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+use function spiralWebDb\CornerstoneTours\render_the_tour_regions;
+use function spiralWebDb\CornerstoneTours\render_tour_comments;
+
+/**
+ * @covers ::\spiralWebDb\CornerstoneTours\Template\render_postmeta_before_content
+ *
+ * @group tours
+ * @group template
+ */
+class Tests_RenderPostmetaBeforeContent extends Test_Case {
+
+	/**
+	 * @dataProvider addTestData
+	 */
+	public function test_should_render_postmeta_before_entry_content( $post_data, $post_meta, $expected_html ) {
+		$post    = $this->factory->post->create_and_get( $post_data );
```

Comment:
For this test, you only need the `post_id` and not the entire object. You can simple the code by using the `create()` method which returns the post’s ID.

```php
$tour_id = $this->factory->post->create( $post_data );
```